docker/utils/container_interface.py

```diff
@@ -100,22 +114,23 @@ def start(self):
             " background...\n"
         )
 
-        # build the image for the base profile
-        subprocess.run(
-            [
-                "docker",
-                "compose",
-                "--file",
-                "docker-compose.yaml",
-                "--env-file",
-                ".env.base",
-                "build",
-                "isaac-lab-base",
-            ],
-            check=False,
-            cwd=self.context_dir,
-            env=self.environ,
-        )
+        # build the image for the base profile if not running base (up will build base already if profile is base)
+        if self.profile != "base":
```

Comment:
This is the mentioned check to not double build-effort if the profile is "base", since the subsequent docker compose up command has --rebuild set to true, so it will get built anyways if profile is "base".